mozilla/activity-stream
# bug 1483655 - Add a new explainer to the Recommended by Pocket section

Comps: https://mozilla.invisionapp.com/share/AMNI4YWRJK6#/screens/314216120_pocket-Section-Disclaimer

To test:

1. Go to about:config and change browser.newtabpage.activity-stream.pocketCta to this: `{"cta_button":"","cta_text":"","cta_url":"","use_cta":true}` by making those strings empty, it by default uses the l10n'd version. If we run a shield study on the copy here, we would need to target a certain locale with it but that to my knowledge is fine.

1. Go to about:home and you should see:
![loggedout](https://user-images.githubusercontent.com/197334/44431992-380a8800-a56e-11e8-8a6f-9025f9270bad.png)

2. now log into pocket, and you should see the same thing, but topics replacing the get pocket cta.

You can also test the copy in the cta using the pref "pocketCta"
